examples/sri-1/components/workgroup/workgroup/workgroup.interface.ts

```diff
@@ -0,0 +1,34 @@
+import { Workstep } from '../workstep/workstep';
+import { BpiSubject } from '../identity/bpiSubject';
+import { Security } from '../policy/security';
+import { Privacy } from '../policy/privacy';
+import { Workflow } from '../workflow/workflow';
+
+export interface IWorkgroup {
+    id: string; // TODO: Add uuid after #491
+    name: string;
+    administrator: BpiSubject[];
+    securityPolicy: Security;
+    privacyPolicy: Privacy;
+    participants: BpiSubject[];
+    worksteps: Workstep[];
+    workflows: Workflow[];
+
+    addParticipant(bpiSubject: BpiSubject);
+    addParticipants(bpiSubjects: BpiSubject[]);
+    getParticipantById(id: string): BpiSubject;
+    getAllParticipants(): BpiSubject[];
+    updateParticipant(id: string, update:any): BpiSubject;
+    removeParticipant(id: string): BpiSubject;
+    removeParticipants(ids: string[]): BpiSubject[];
+    addSecurityPolicy(securityPolicy: Security);
+    removeSecurityPolicy(securityPolicy: Security);
+    updateSecurityPolicy(id: string, ...udpates: string[]);
+    addPrivacyPolicy(privacyPolicy: Privacy);
+    removePrivacyPolicy(privacyPolicy: Privacy);
+    updatePrivacyPolicy(id: string, ...udpates: string[]);    
+    addWorkstep(workstep: Workstep);
+    getWorkstepById(workstepId: string);
```

Comment:
@Kasshern see above to use arrays for multiples

Reply:
Altered to use array.